CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# 1.0.3
+
+* Implement more I/O safety traits
+  (https://github.com/tokio-rs/mio/pull/1831).
+* Remove hermit-abi dependency, now using libc
+  (https://github.com/tokio-rs/mio/pull/1830).
+* Use `poll(2)` implementation on AIX, removing the need for using
+  `mio_unsupported_force_poll_poll`
+  (https://github.com/tokio-rs/mio/pull/1833).
```

Comment:
Up to you, but I'd have made a minor release rather than a patch release for this.

Reply:
Maybe, I don't really care about minor vs. patch, as long as it's not breaking :)

Reply:
Fair. For Mio we don't have that many changes so I haven't needed it yet.